apps/prairielearn/src/pages/instructorAssessmentQuestions/components/InstructorAssessmentQuestionsTable.tsx

```diff
@@ -3,41 +3,52 @@ import { useState } from 'preact/hooks';
 import { Fragment } from 'preact/jsx-runtime';
 import { Dropdown } from 'react-bootstrap';
 
+import { run } from '@prairielearn/run';
+
 import { AssessmentBadge } from '../../../components/AssessmentBadge.js';
-import {
-  AssessmentQuestionHeaders,
-  AssessmentQuestionNumber,
-} from '../../../components/AssessmentQuestions.js';
+import { AlternativeGroupHeader, ZoneHeader } from '../../../components/AssessmentQuestions.js';
 import { HistMini } from '../../../components/HistMini.js';
 import { IssueBadge } from '../../../components/IssueBadge.js';
 import { SyncProblemButton } from '../../../components/SyncProblemButton.js';
 import { TagBadgeList } from '../../../components/TagBadge.js';
 import { TopicBadge } from '../../../components/TopicBadge.js';
 import type { StaffAssessmentQuestionRow } from '../../../lib/assessment-question.js';
 import type { StaffCourse } from '../../../lib/client/safe-db-types.js';
+import type { Assessment, AssessmentQuestion, EnumAssessmentType } from '../../../lib/db-types.js';
 import { idsEqual } from '../../../lib/id.js';
-import { assertNever } from '../../../lib/types.js';
+import type {
+  QuestionAlternativeJson,
+  ZoneAssessmentJson,
+  ZoneQuestionJson,
+} from '../../../schemas/infoAssessment.js';
 
+import { EditQuestionModal } from './EditQuestionModal.js';
 import { ExamResetNotSupportedModal } from './ExamResetNotSupportedModal.js';
 import { ResetQuestionVariantsModal } from './ResetQuestionVariantsModal.js';
 
 function Title({
   questionRow,
   hasCoursePermissionPreview,
   urlPrefix,
+  questionNumber,
+  alternativeNumber,
 }: {
   questionRow: StaffAssessmentQuestionRow;
   hasCoursePermissionPreview: boolean;
   urlPrefix: string;
+  questionNumber: number;
+  alternativeNumber?: number;
 }) {
-  const { question, assessment_question, alternative_group, alternative_group_size } = questionRow;
+  const { question } = questionRow;
   const title = (
     <>
-      <AssessmentQuestionNumber
-        assessmentQuestion={assessment_question}
-        alternativeGroup={alternative_group}
-        alternativeGroupSize={alternative_group_size}
-      />
+      {alternativeNumber !== undefined ? (
+        <span class="ms-3">
+          {questionNumber}.{alternativeNumber + 1}.{' '}
+        </span>
+      ) : (
+        `${questionNumber}. `
+      )}
```

Comment:
Nit, it seems like a minor regression to switch from `<AssessmentQuestionNumber>` to inlining this - can you say more about that?